jetstream/statistics.py

```diff
@@ -304,7 +328,11 @@ def _decilize(arr):
         return arr_dict
 
     def transform(
-        self, df: DataFrame, metric: str, reference_branch: str
+        self,
+        df: DataFrame,
+        metric: str,
+        reference_branch: str,
+        _experiment: "config.ExperimentConfiguration",
```

Comment:
The parameter names are part of the interface so this should still be `experiment` here and the other cases